__tests__/app.test.js

```diff
@@ -269,4 +269,90 @@ describe("POST /api/articles/:article_id/comments",()=>{
   
   })
 
-})
\ No newline at end of file
+})
+
+
+describe("PATCH /api/articles/:article_id",()=>{
+  test("200:Responds with incremented patched object ",()=>{
+    return request(app)
+    .patch("/api/articles/1")
+    .send({ inc_votes: 5 })
+    .expect(200)
+    .then(({body:{article}})=>{
+      expect(article).toMatchObject( {
+        article_id: 1,
+        title: 'Living in the shadow of a great man',
+        topic: 'mitch',
+        author: 'butter_bridge',
+        body: 'I find this existence challenging',
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 105,
+        article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+      })
+    })
+  })
```

Comment:
This is a really good test, but some of these values can be simply asserted as strings for easier readability